pkg/gits/bitbucket.go

```diff
@@ -389,6 +393,16 @@ func (b *BitbucketCloudProvider) UpdatePullRequestStatus(pr *GitPullRequest) err
 	return nil
 }
 
+func (p *BitbucketCloudProvider) GetPullRequest(owner, repo string, number int) (*GitPullRequest, error) {
```

Comment:
not a biggie but seen as GetPullRequest is a simple helper method wrapping UpdatePullRequestStatus we could just add it as a generic helper method - not a biggie though

Reply:
This is your baby, it just seems much more intuitive to have a `GetPullRequest`?

Reply:
I was just leaning towards the poor GitProvider implementor having more work to do ;) Not a biggie though